mobile/mobile.go

```diff
@@ -574,8 +574,8 @@ func (m *Mobile) getImageData(id string, path string, isThumb bool) (string, err
 	// get meta data for url type
 	meta, err := thrd.GetPhotoMetaData(id, block)
 	if err != nil {
-		log.Errorf("get photo meta data failed %s: %s", id, err)
-		return "", err
+		log.Warningf("get photo meta data failed %s: %s", id, err)
+		meta = &model.PhotoMetadata{}
```

Comment:
same deal here, definitely want to just guess the image type of we can't find meta... have a feeling this is causing some `TextileImage` related crashes